editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2556,7 +2556,7 @@ void ThemeTypeEditor::_update_type_items() {
 			HBoxContainer *item_control = _create_property_control(Theme::DATA_TYPE_FONT, E.key, E.value);
 			EditorResourcePicker *item_editor = memnew(EditorResourcePicker);
 			item_editor->set_h_size_flags(SIZE_EXPAND_FILL);
-			item_editor->set_base_type("Font");
+			item_editor->set_base_type("FontItem");
```

Comment:
Is this correct? This should be a type not the icon no?

Reply:
Looks like a search and replace error, I know all about them myself!

Reply:
```suggestion
			item_editor->set_base_type("Font");
```

Reply:
> Looks like a search and replace error, I know all about them myself!

No, I was just reckless.